rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -615,7 +617,7 @@ impl Controller {
                 if let Some(req) = self.auth.start_sign_in()? {
                     let url = req.to_url(&self.advanced_settings.auth_base_url);
                     self.refresh_system_tray_menu()?;
-                    os::open_url(&self.app, &url)?;
+                    tauri::api::shell::open(&self.app.shell_scope(), url.expose_secret(), None)?;
```

Comment:
This isn't related, but it was in the same files and I noticed I could de-duplicate it.